packages/stateless/components/FormattedJsonDisplay.tsx

```diff
@@ -19,7 +21,7 @@ export const FormattedJsonDisplay = ({
   ) : jsonLoadable.state === 'hasValue' &&
     jsonLoadable.contents !== undefined ? (
     <div className="space-y-2">
-      <InputLabel name={t('form.tokenInfo')} />
+      <InputLabel name={title ?? t('form.tokenInfo')} />
```

Comment:
<3 probably don't even want the `tokenInfo` default here, but that is tech debt for later :P